webui/react/src/pages/Cluster/ClusterHistoricalUsage.tsx

```diff
@@ -103,66 +104,68 @@ const ClusterHistoricalUsage: React.FC = () => {
 
   return (
     <div className={css.base}>
-      <Space align="end" className={css.filters}>
-        <ClusterHistoricalUsageFilters value={filters} onChange={handleFilterChange} />
-        <Button onClick={() => setIsCsvModalVisible(true)}>Download CSV</Button>
-      </Space>
-      <Section bodyBorder loading={!chartSeries} title="Compute Hours Allocated">
-        {chartSeries && (
-          <ClusterHistoricalUsageChart
-            groupBy={chartSeries.groupedBy}
-            hoursByLabel={chartSeries.hoursTotal}
-            time={chartSeries.time}
+      <SyncProvider>
```

Comment:
these diffs are really annoying. the only change was that i wrapped everything with `SyncProvider`